.travis.yml

```diff
@@ -7,4 +7,5 @@ before_install:
   - sh -e /etc/init.d/xvfb start
 script:
   - npm install
+  - pub run dart_dev test --unit
```

Comment:
I thought so too, but didn't see them being run in the CI output for commit f235d0c
https://travis-ci.org/Workiva/sockjs-dart-client/builds/129480139